lib/db/device_files_db.dart

```diff
@@ -34,6 +37,7 @@ extension DeviceFiles on FilesDB {
           {
             "id": localID,
             "path_id": pathID,
+            "synced": syncStatus ? _sqlBoolTrue : _sqlBoolFalse
```

Comment:
`isTracked` is the best option I have :(

Reply:
Can fix one after tomorrow's discussion

Reply:
removed this local state.